kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -138,7 +145,7 @@
           this.content.is_leaf +
           (this.isUserLoggedIn && !this.isLearner && this.content.num_coach_contents) +
           (this.content.num_coach_contents > 0) +
-          (this.content.copies_count > 1) +
+          (this.content.copies > 1) +
```

Comment:
`content.copies` is an array.
```suggestion
          (this.content.copies && this.content.copies.length > 1) +
```